### DISCUSSION TITLE
Outline server cpu 100% Utilization rate

### DISCUSSION THREAD

**Discussion by Dominicus0830** [outline/outline#5025] [Self Hosting]
![Screenshot_20230313_095214_Chrome](https://user-images.githubusercontent.com/100403377/224590668-53ee96aa-a9ed-43d8-b6f2-a591602a5487.jpg)

**Comment by Dominicus0830** [comment]
How to solve that

**Comment by tommoor** [comment]
Can't say I'm clear on what these partial screenshots are showing. You should post some logs of what the server is doing at 100% CPU.

**Reply by Dominicus0830** [reply]
Please tell me the path of the log file.

**Reply by tommoor** [reply]
Outline logs to stdout/stderr, I couldn't say where you would be storing those logs
